src/bash_autocomplete.sh

```diff
@@ -4,14 +4,14 @@ function _kw_autocomplete()
   COMPREPLY=()
   current_command="${COMP_WORDS[COMP_CWORD]}"
   previous_command="${COMP_WORDS[COMP_CWORD - 1]}"
-  kw_options="explore e build b bi init new n ssh s clear-cache
-                mount mo umount um vars up u codestyle c configm g
+  kw_options="explore e bd build b init ssh s clear-cache
+                mount mo umount um vars v up u codestyle c configm g
```

Comment:
`kw v` was dropped since the introduction of `kw -v`! See commit `f8af2abeae4265cc0d55fd9f3544427b02ffadd1` and pull request #169.